app/src/main/java/com/kickstarter/libs/utils/AnimationUtils.kt

```diff
@@ -78,4 +78,17 @@ object AnimationUtils {
     crossFadeAndReverseAnimatorSet.playSequentially(startAnimation, endAnimation)
     return crossFadeAndReverseAnimatorSet
   }
+
+  fun notificationBounceAnimation(view: View, secondView: View) {
+    val pvhX = PropertyValuesHolder.ofFloat(View.SCALE_X, 1.0f, 1.3f, 1.0f)
```

Comment:
Should these values be constants because we reuse them? Maybe `INITIAL_SCALE` and `MAX_SCALE`?